internal/clients/grpc/connect.go

```diff
@@ -84,8 +85,13 @@ func Setup(config Config, svcName, jaegerURL string) (*Client, ClientHandler, er
 		return nil, nil, errors.Wrap(errGrpcConnect, err)
 	}
 
+	jURL, err := url.Parse(jaegerURL)
+	if err != nil {
```

Comment:
Nice to have this check 

Could you return error with error message something like below code 
```go 
	jURL, err := url.Parse(jaegerURL)
	if err != nil {
		return nil, nil, errors.Wrap(ErrFailedToParseJagerUrl,err)
	}
```